Dockerfile-tests

```diff
@@ -9,6 +9,21 @@ COPY project /app/project
 COPY build.sbt /app
 COPY automation-tests-docker-entrypoint.sh /app
 
+RUN touch /app/key.json
+
+# Downloading gcloud package
+RUN curl https://dl.google.com/dl/cloudsdk/release/google-cloud-sdk.tar.gz > /tmp/google-cloud-sdk.tar.gz
+
+# Installing the package
+RUN mkdir -p /usr/local/gcloud \
+  && tar -C /usr/local/gcloud -xvf /tmp/google-cloud-sdk.tar.gz \
+  && /usr/local/gcloud/google-cloud-sdk/install.sh
+
+# Adding the package path to local
+ENV PATH $PATH:/usr/local/gcloud/google-cloud-sdk/bin
+
+RUN mkdir -p /root/.ssh
+
```

Comment:
This looks to be from here:
https://stackoverflow.com/questions/28372328/how-to-install-the-google-cloud-sdk-in-a-docker-image

... and adding the [official documentation](https://cloud.google.com/sdk/docs/install#linux) (for posterity's sake)

Reply:
Yeah, I'll add a comment. Naturally, the official docs didn't work and I had to use the stack overflow docs